src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja

```diff
@@ -100,6 +100,11 @@
                 {% if specter.wallet_manager.failed_load_wallets %}
                     <form action="{{ url_for('wallets_endpoint.failed_wallets') }}" method="POST">
                         <input type="hidden" class="csrf-token" name="csrf_token" value="{{ csrf_token() }}"/>
+                        {% if specter.node.alias == "spectrum_node" %}
+                            <div class="p-4 bg-dark-600 rounded-lg mt-2 mb-2" role="alert">
+                                {{ _('Wallets created with Bitcoin Core need to be recreated when using Spectrum. Click "Show Details" to do this.') }}
+                            </div>
+                        {% endif %}
```

Comment:
It would be good to put this into a node specific template, not directly in the sidebar. So maybe we can have an include here referencing to a specific template, something like:
```
{% include specter.node.node_wallet_failed_loading_remark() %}
```